packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -180,6 +181,10 @@ async function generateExamplePages(pageSection, docsPath, sourceDir, options, c
 module.exports = async function generatePages(sourceDir, docsDir, options, changedPath) {
   logTask('📝 ', 'Generating documentation pages');
 
+   // Create HTML files for example and doc pages
+  const docsPath = path.join(docsDir, 'dist');
```

Comment:
I'm not sure if this comment is accurate, `docsPath` is just the location of all the build files including HTML, CSS and JS files. I would personally move this variable declaration right before [this line](https://github.com/CMSgov/design-system/pull/890/files#diff-9af9efe78a2334efeb563a0d95bd8b68a4f0187123d4ad7d875ecde2ad3403d1R235), as its not actually used until `generateExamplePages` and `generateDocPages`